in_memory_person_storage.go

```diff
@@ -21,9 +21,58 @@ func (s *InMemoryPersonStorage) Add(person *Person) error {
 }
 
 func (s *InMemoryPersonStorage) GetAll() []*Person {
-	persons := make([]*Person, 0)
+	var persons []*Person
 	for _, person := range s.data {
 		persons = append(persons, person)
 	}
 	return persons
 }
+
+func (s *InMemoryPersonStorage) GetPersonById(id uuid.UUID) *Person {
+	p, ok := s.data[id]
+	if !ok {
+		return nil
+	}
+	return p
+}
+
+func (s *InMemoryPersonStorage) GetPersonsByName(name string) []*Person {
```

Comment:
in methods that return array it is common to return empty slice if no data was found, but still have the second returned error parameter